### DISCUSSION TITLE
Feature Request: Implement `flux:prose` for Tailwind Prose Typography

### DISCUSSION THREAD

**Discussion by realpoke** [livewire/flux#481] [Feature Requests]
## Description

I'd like to propose a feature that allows us to easily implement the Tailwind Prose typography.

### Background

The Tailwind Prose typography feature, as outlined in the [Tailwind CSS Typography](https://github.com/tailwindlabs/tailwindcss-typography) repository, provides a set of utilities for beautifully styled text. Since Flux is built on top of Tailwind CSS, it would be beneficial to have a straightforward way to apply these typography styles using a `flux:prose` directive. Since the typography from tailwindcss does not match Flux exactly, and needs small adjustments in the styling.

### Proposed Solution

- Implement a `flux:prose` directive that applies the necessary Tailwind Prose styles and has minor adjustments to align with Flux.
- Ensure that this directive leverages the existing prose system in Tailwind, making any adjustments needed to fit seamlessly within the Flux framework.

### Benefits

- Simplifies the process of applying typography styles.
- Enhances the overall aesthetic of text-heavy components in Flux applications.
- Follows the conventions already established in Tailwind CSS, making it intuitive for developers familiar with the framework.
- Easy markdown rendering/styling.

This feature should be relatively straightforward to implement, given the foundation laid by the Tailwind CSS Typography plugin. Looking forward to your thoughts on this proposal!

**Comment by realpoke** [comment]
I think this is even more relevant now when we got the rich text editor. To show the text from the database.

**Comment by nicolaslefresne** [comment]
Seems like a pretty stupid question, I'm sorry, but how do you display a text on your page that is the exact visual replica of what you have in the flux editor?

Because using Tailwind prose class kinda works, but does not really reproduces the font size/space/etc, am I missing something obvious here?

**Reply by realpoke** [reply]
This is why I propose a custom flux:prose that will have custom a Tailwind setup.
